File: playify/lib/src/class/artist/artist.dart
import 'package:flutter/material.dart';
import 'package:playify/src/class/album/album.dart';

class Artist {
  Artist({required this.name, required this.albums});

  ///The name of the artist.
  String name;

  ///The albums of the artist.
  List<Album> albums;

  @override
  String toString() {
    return 'Name: ' + name + ' has ' + albums.length.toString() + ' albums\n';
  }
}

File: playify/android/src/main/kotlin/com/kaya/playify/playify/Classes/Artist.kt
package com.kaya.playify.playify.Classes

class Artist {
    constructor(name: String?, albums: List<Album>?) {
        this.name = name
        this.albums = albums
    }

    ///The name of the artist.
    var name: String? = null

    ///The albums of the artist.
    var albums: List<Album>? = null

    override fun toString(): String {
        return "Artist(name=$name, albums=$albums)"
    }
}